src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,51 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private final int figuresCount = 3;
+
```

Comment:
redundant blank line

Reply:
already changed the structure of file, I hope it fixed